scripts/gh/overview-dag.sh

```
@@ -0,0 +1,123 @@
+# Parse "Blocked by" lines in bodies (best-effort)
+all_nums=$(jq -r '.[].number' "$tmpdir/issues.json")
+for n in $all_nums; do
+  body=$(gh issue view --repo "$REPO" "$n" --json body --jq .body 2>/dev/null || echo "")
+  mapfile -t lines < <(printf '%s
+' "$body" | grep -i "Blocked by" || true)
+  for l in "${lines[@]}"; do
+    # Extract all #<num> tokens on that line
+    while [[ "$l" =~ \#([0-9]+) ]]; do
+      blk="${BASH_REMATCH[1]}"
+      echo "block $n <- $blk" >> "$tmpdir/edges.txt"
+      l="${l#*#${blk}}" # advance
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)